swan-cmd/src/main/java/de/fraunhofer/iem/swan/data/Method.java

```diff
@@ -38,6 +38,7 @@ public class Method {
     private Set<Category> srm;
     private Set<Category> cwe;
     private boolean known;
+    private boolean isApplicationMethod;
```

Comment:
Why are you adding this field in this class? This class is used to generate the JSON file and this information is not something that should be stored in the file.

Reply:
JsonIgnore added.